sql/ddl/alter/sql-practice-alter-table-2.md

```diff
@@ -23,7 +23,7 @@ standards:
 
 
 links:
-  - '[MySQL modify columns documentation](https://dev.mysql.com/doc/refman/5.7/en/alter-table.html){website}'
+  - '[MySQL modify columns documentation](https://dev.mysql.com/doc/refman/5.7/en/alter-table.html){Documentation}'
```

Comment:
probably we want it with lowercase, not sure it's automatically done before going in the db